examples/delegated-routing/package.json

```diff
@@ -3,19 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "ipfs": "~0.34.4",
-    "libp2p": "github:libp2p/js-libp2p#master",
-    "libp2p-delegated-content-routing": "~0.2.2",
-    "libp2p-delegated-peer-routing": "~0.2.2",
-    "libp2p-kad-dht": "~0.14.12",
-    "libp2p-mplex": "~0.8.5",
-    "libp2p-secio": "~0.11.1",
-    "libp2p-webrtc-star": "~0.15.8",
-    "libp2p-websocket-star": "~0.10.2",
-    "libp2p-websockets": "~0.12.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-scripts": "2.1.8"
+    "ipfs": "~0.40.0",
```

Comment:
We need to get `libp2p` to `js-ipfs` before getting this to work. 

However, I could also not test this with #master . Do we need to run `go-ipfs` in the specific branch mentioned in the README?

Reply:
There are some configuration issues that need to be resolved, see https://github.com/libp2p/js-libp2p/pull/371#pullrequestreview-255991314. 

In regards to requiring `ipfs`, we may want to revisit this example and only have libp2p in the client. The example cats a file right now, but in reality we just need to find things on the network via the delegate node.

Reply:
For this example, should we have a `js-libp2p` node running and providing a few cid that we can then find with the app example?